package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "dev": "yarn run app:dev",
     "app:dev": "yarn tauri dev --config bin/tauri.dev.conf.json --features debug --target $(yarn run rust-target)",
-    "app:build": "yarn tauri build --target $(yarn run rust-target)",
+    "app:build": "yarn run app:build:nobundle && ./scripts/postbuild.sh && yarn run app:build:bundle",
+    "app:build:nobundle": "yarn tauri build --no-bundle --target $(yarn run rust-target)",
+    "app:build:bundle": "yarn tauri bundle --bundles app,dmg --target $(yarn run rust-target)",
```

Comment:
this is not used during the release process

we need to check the release action and see how we can configure it to run the script
https://github.com/tauri-apps/tauri-action

from a quick look, the `tauriScript` option might be what we need

Reply:
It wasn't since the tauri-action only runs the `tauri build` command in a single step. Ended up finding about the `beforeBundleCommand` hook to simply run the script between the build and bundle steps